it/src/main/java/org/corfudb/universe/scenario/fixture/Fixtures.java

```diff
@@ -64,15 +66,16 @@ private TestFixtureConst() {
         public static final int DEFAULT_WAIT_POLL_ITER = 300;
 
         // Default time to wait before next layout poll: 1 second
-        public static final int DEFAULT_WAIT_TIME = 1;
+        public static final Duration DEFAULT_WAIT_TIME = Duration.ofSeconds(1);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4663789)